torch/quantization/qconfig.py

```diff
@@ -93,6 +93,10 @@ def __new__(cls, activation=torch.nn.Identity, weight=torch.nn.Identity):
 default_activation_only_qconfig = QConfig(activation=default_fake_quant,
                                           weight=torch.nn.Identity)
 
+# QAT config that uses a fused observer + fake quant modules for optimized training performance.
+# to modify the activation/weight observers, the default entries in fake_quantize.py can be modified.
+default_qat_qconfig_v2 = QConfig(activation=default_fused_act_fake_quant, weight=default_fused_wt_fake_quant)
```

Comment:
Lets have a new version in the get_default_qat_qconfig function, this way we can support both fbgemm and qnnpack and also roll this out easily by bumping the version number
``` python
# Move version to 1 when we want to roll this out.
def get_default_qat_qconfig(backend='fbgemm', version=None):
    # Histogram observer is too slow for quantization aware training

    if backend == 'fbgemm':
      if version is None:
          qconfig = QConfig(activation=FakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
                                                            quant_min=0,
                                                            quant_max=255,
                                                            reduce_range=True),
                          weight=default_per_channel_weight_fake_quant)
      elif backend == 'qnnpack':
          qconfig = QConfig(activation=FakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
                                                            quant_min=0,
                                                            quant_max=255,
                                                            reduce_range=False),
                          weight=default_weight_fake_quant)
 if version ==1:
  if backend =='fbgemm':
      qconfig = QConfig(activation=default_fused_act_fake_quant, weight=default_fused_wt_fake_quant)
if backend=='qnnpack':
    ....
    else:
        qconfig = default_qat_qconfig
    return qconfig
```